docs/plugins/lyrics.rst

```diff
@@ -166,6 +166,16 @@ library. Install it by typing::
 
 The backend is enabled by default.
 
+Activate Tekstowo.pl Lyrics
```

Comment:
Maybe just merge this with the previous section, because it gives the same advice as for the Genius backend?